docs/getting-started/products/ybribe/overview.md

```diff
@@ -0,0 +1,25 @@
+# Overview
+
+
+## What is yBribe?
+
+[yBribe](https://ybribe.yearn.farm) is a platform where vote-escrowed Curve (veCRV) holders can receive compensation from buyers interested in increasing CRV emissions to their Curve pool’s gauge.
+
+In DeFi, incentives for voting are commonly referred to as ‘bribes’. yBribe enables veCRV holders — be they protocols, DAOs, or individual users, to extract the value of their voting power. 
+
+![](https://i.imgur.com/WedS2iP.png)
+
+**yBribe pairs users looking to buy votes, with those looking to sell them.**
+
+Curve conducts weekly gauge votes that determine the allocation of CRV rewards to various pools. By buying votes, DAOs, protocols, and users can influence the direction of these rewards while boosting yields in pools beneficial to them. To do this, use the ‘Bribe a Gauge’ function on yBribe. Once the bribe is posted, voters will see the pending APR for that gauge increase.
```

Comment:
Where is the ‘Bribe a Gauge’ function? I couldn't find it on the website. If it has a different name, I think it's better to share the exact name that it has on yBribe